bin/dwishellmath

```diff
@@ -63,6 +65,9 @@ def execute(): #pylint: disable=unused-variable
     # make a 4D image with one volume
     app.warn('Only one unique b-value present in DWI data; command mrmath with -axis 3 option may be preferable')
     run.command('mrconvert ' + files[0] + ' ' + path.from_user(app.ARGS.output) + ' -axes 0,1,2,-1', mrconvert_keyval=path.from_user(app.ARGS.input, False), force=app.FORCE_OVERWRITE)
+  
+  if('mrtrix-tmp-' in app.ARGS.output):
+    sys.stdout.write(app.ARGS.output)
```

Comment:
This is leftover from initial attempts to get output piping working; a more general solution will be required.